app/build.gradle

```diff
@@ -206,7 +206,14 @@ dependencies {
     implementation 'com.amazonaws:aws-android-sdk-core:2.16.8'
     implementation 'com.amazonaws:aws-android-sdk-s3:2.16.8'
 
-    implementation 'org.koin:koin-android-viewmodel:2.0.0-GA'
+    //koin dependencies
+    implementation "io.insert-koin:koin-core:$koin_version"
+    implementation "io.insert-koin:koin-android:$koin_version"
```

Comment:
Do we need both of these? I'm curious if the android version contains core in it already

Reply:
Not both. I just realised Koin android alone is enough for the project.